xpdacq/calib.py

```diff
@@ -40,11 +41,14 @@ def _check_obj(required_obj_list):
                             "namespace".format(obj_str))
     return
 
+
 def _timestampstr(timestamp):
-    ''' convert timestamp to strftime formate '''
-    timestring = datetime.datetime.fromtimestamp(float(timestamp)).strftime('%Y%m%d-%H%M')
+    """ convert timestamp to strftime formate """
+    timestring = datetime.datetime.fromtimestamp(float(timestamp)).strftime(
+        '%Y%m%d-%H%M')
```

Comment:
This is how PEP8 has it: http://legacy.python.org/dev/peps/pep-0008/#indentation